openedx/core/djangoapps/course_live/views.py

```diff
@@ -0,0 +1,114 @@
+"""
+View for course live app
+"""
+from typing import Dict
+
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from lti_consumer.api import get_lti_pii_sharing_state_for_course
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.serializers import ValidationError
+from rest_framework.views import APIView
+
+from common.djangoapps.util.views import ensure_valid_course_key
+from openedx.core.djangoapps.course_live.permissions import IsStaffOrInstructor
+from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
+
+from .models import AVAILABLE_PROVIDERS, CourseLiveConfiguration
+from .serializers import CourseLiveConfigurationSerializer
+
+
+class CourseLiveConfigurationView(APIView):
+    """
+    View for configuring discussion settings.
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser
+    )
+    permission_classes = (IsStaffOrInstructor,)
+
+    @ensure_valid_course_key
+    def get(self, request: Request, course_key_string: str) -> Response:
+        """
+        Handle HTTP/GET requests
+        """
+        pii_sharing_allowed = get_lti_pii_sharing_state_for_course(course_key_string)
+        if not pii_sharing_allowed:
+            return Response({
```

Comment:
If we can have a dedicated status code for this Response. It will be easier to handle on the frontend

Reply:
I don't think It would be required as UI must be rendered based  on `pii_sharing_allowed` boolean